.github/buildomat/jobs/build-and-test-linux.sh

```diff
@@ -3,7 +3,7 @@
 #: name = "build-and-test (ubuntu-20.04)"
 #: variety = "basic"
 #: target = "ubuntu-20.04"
-#: rust_toolchain = "nightly-2022-09-27"
+#: rust_toolchain = "stable-1.66.1"
```

Comment:
These should just be `1.66.1`. The format is `<channel>[-<date>][-<host>]` with `<channel>` being `stable|beta|nightly|<major.minor>|<major.minor.patch>`. You can't combine the version and stable like that.

https://rust-lang.github.io/rustup/concepts/toolchains.html#toolchain-specification

Reply:
Good call.  Not sure how I didn't notice this in previous testing.